### DISCUSSION TITLE
Final Exam: Questions/Concerns About Difficulty, Number of Attempts, etc.

### DISCUSSION THREAD

**Discussion by JamesDBartlett3** [TabularEditor/TabularEditor3#1267] [Learn]
Hello!

I recently completed all of the Tabular Editor Learn courses, but I'm reluctant to take the final exam for a number of reasons. 

During the courses, I got most of the quiz questions wrong on the first attempt (and often even on the second attempt as well), even though I was paying very close attention to the course material, and I could easily explain why I answered the way I did. I felt that most of the quiz questions were ambiguous/vague, set up deceptively/unconventionally (e.g. allowing only one correct answer on a multi-select question, etc.), and sometimes even directly contradictory of the course material. I now wish that I had taken the time to document these issues with the quiz questions as I progressed through the courses, but due to the lengthy nature of the courses, I thought that if I took the time to document every time I had an issue with a quiz question, it would be months before I could take the final exam.

I'm now wondering whether the final exam is considered to be similar in difficulty to the quiz questions in the courses, and what will happen if I am unable to pass the final exam within 3 attempts. I'd also like to know whether I will be given an explanation for why my answer was incorrect, either during or after an attempt on the exam, so that I may make the necessary adjustments to my notes before taking another attempt. 

Thank you,

James

**Comment by mlonsk** [comment]
Hi @JamesDBartlett3 
First off, well done on completing the courses! That is no small feat!
I am sorry to hear that the questions felt unaligned with the course material, would you be able to provide an example of a question where this happened to you so we can evaluate our courses going forward?

The exam questions are in the same style, difficulty and covers the content of all the courses. There are 25 questions chosen randomly from a pool of question for each attempt and the time limit for the exam is 3 hours so you have time to research your answers. Most of the questions are single select, but there are multiple select among them.
Once finished you will be presented with your score and you can review your answers, but you will however not be given explanations to the answers or which answer was the correct one. The passing grade is 80 % correct answers. 
If you fail all three times you will not be able to redo the exam again. 

I hope this has given you some clarification and I think you are able to pass the exam as you sound like you have been very diligent in your studies. The best advice I can give is to really read the questions and answers options very well as well as having Tabular Editor open at the same time so you can test out specific questions.

**Comment by JamesDBartlett3** [comment]
@mlonsk, 

> would you be able to provide an example of a question where this happened to you so we can evaluate our courses going forward?

I didn't make note of the questions that seemed misaligned with the course content while I was going through the course, mainly because my employer was paying for my time to complete this training. Since the course material alone took close to a week's time for me to complete it, I didn't feel like it would be fair to my employer if I took even more time to report my observations.

> The passing grade is 80 % correct answers. If you fail all three times you will not be able to redo the exam again.

To be perfectly honest, this is **_extremely harsh_** grading criteria compared to most standard certification exams in the BI/Data industry. For example, Microsoft's certification exams (e.g. PL-300, DP-600, etc.) are considered quite rigorous by industry standards, and [they can be re-taken as many times as necessary](https://learn.microsoft.com/en-us/credentials/support/retake-policy) to achieve [a passing grade of 70% or higher](https://learn.microsoft.com/en-us/credentials/certifications/exam-scoring-reports#scores-needed-to-pass-exams). Can you comment on why you chose to make the criteria for this exam so extreme compared to other certification exams in the BI/Data industry?

> I think you are able to pass the exam as you sound like you have been very diligent in your studies.

I wish I could say that I felt the same. As it stands currently, I have zero confidence that I would pass the final exam on my first, second, or third attempt. Therefore, unless you're willing to consider relaxing the "three attempts" rule, I'll just have to go through all of the courses once again, on my own time, and submit a ticket for every ambiguous, vague, deceptive, or otherwise problematic quiz question that I encounter. 

James

**Comment by Emma123467** [comment]
Hello, I am wondering if there are many users passing this exam within 1-3 attempts? I find this exam very difficult and I also do not always find explicit answers to the questions in the training materials. I am not sure what are the questions I answered correctly especially when there is a multiple choice or true/false statements. Is there anyway for me to find out what I answered correctly and what not? Can this info be provided after an attempt? This could be a good learning experience. I also agree that 80% passing grade is very high. Thank you for your support.

**Comment by Emma123467** [comment]
Example question what is the first thing you to look at when you have issues in the model? The possible answers are BPA Analyzer, Vertipaq, DAX Optimizer and something more. Does this really speak about my skills if I look at all of them or in another order than recommended by you? I guess the goal is to optimize and to know how to use it. Another question about feedback to another developer -  one of the answers is he should never use calculated columns. I agree there is a recommendation not to use calculated columns in general but there could be case when it is needed. I do not know anymore what author intends here specifically with this question and answer "NEVER" applies or it does not. I find it very confusing. The way some of the questions and answers are defined is subject to interpretation which are not necessarily an indicator of my skills and knowledge but maybe the authors preferences.

**Reply by otykier** [reply]
Hi @Emma123467 

I understand that failing the exam can be frustrating. We deliberately wanted to create an exam that is above average difficulty, for a couple of reasons: First of all, the exam isn't proctored/monitored, and you have 3 hours to answer 25 questions. Also, we wanted to make sure that getting the certificate is truly an achievement - something you can be proud of, and not just something that everyone can easily power through without any effort.

I'm sorry if the questions are too confusing, but we have tried our best to remove any ambiguities or subjectivity from the questions and answers. But as usual, the devil is in the detail. You have to pay close attention to the way the question is formulated.

For example:

> Borp thinks his model contains a lot of problems.
> Borp's looking for a <ins>quick</ins> or <ins>automated way</ins> to identify <ins>common issues</ins>.

If you consider the keywords I highlighted above, the correct answer should be more apparent. While there are many *possible* ways of checking for problems in a model, like you point out, the key thing we're testing here is whether learners understand the difference between these options, and in this case, which option is the quickest/most automated, and most useful for identifying common issues.

For the other example, I can tell you that "Zip should never use Calculated Tables or Calculated Columns." is NOT among the correct answers to this question. As you correctly point out, it is a general recommendation, but never say never!
